main.go

```diff
@@ -94,6 +99,7 @@ func main() {
 	flag.StringVar(&healthAddr, "health-addr", ":9440", "The address the health endpoint binds to.")
 	flag.StringVar(&storagePath, "storage-path", "/data", "Where to store the persistent database of image metadata")
 	flag.Int64Var(&storageValueLogFileSize, "storage-value-log-file-size", 1<<28, "Set the database's memory mapped value log file size in bytes. Effective memory usage is about two times this size.")
+	flag.Uint16Var(&gcInterval, "gc-interval", 1, "The number of minutes to wait between garbage collections. 0 disables the garbage collector.")
```

Comment:
I'm wandering if the 1m interval is too aggressive and if it will increase the CPU usage unnecessarily for most users. How about we go with 5m as the default, and we document [here](https://fluxcd.io/flux/components/image/options/) the flag (those that have a massive amount of images could set it to 1m).

Reply:
After running some load-tests and considering how much smaller image tag data is -- I don't think it's very common for this GC code to run with the default BadgerDB table/LSM/vlog settings. The database has to contain a large number of image tags and use more than the default memory limit of 1GB to start flushing to vlog files.

I've updated the default interval to 10 minutes.
My hypothesis right now is end-users who are seeing this in production either have:
a) a very large number of image repos, or
b) a constantly changing set of repos with taglists larger than 1MB each